components/chainhook-cli/src/scan/bitcoin.rs

```diff
@@ -290,7 +290,7 @@ pub async fn execute_predicates_action<'a>(
             gather_proofs(&trigger, &mut proofs, config, ctx);
         }
         let predicate_uuid = &trigger.chainhook.uuid;
-        match handle_bitcoin_hook_action(trigger, &proofs) {
+        match handle_bitcoin_hook_action(trigger, &proofs, &config.predicates_config) {
```

Comment:
It seems more reasonable to pass the parent `EventObserverConfig` here and let the `handle_bitcoin_hook_action` function decide what it needs. The caller don't need to know about implementation details of the callee function.